### ISSUE TITLE
Issue with sketch plug-in

### ISSUE THREAD

**Issue by j-sharrock** [avocode/avocode#1747]
Sketch plug-in is not available on Sketch. Some sort of malfunction happening. 

---
* **Version**: 2.17.0
* **Sketch plugin version**: undefined
* **Web/Desktop**: Desktop
* **Platform**: darwin
* **Avocode UID**: 218612
---
```
Error: Command failed: "/Applications/Origami Studio.app/Contents/Resources/sketchtool/bin/sketchtool" show plugins
/bin/sh: /Applications/Origami Studio.app/Contents/Resources/sketchtool/bin/sketchtool: No such file or directory

    at ChildProcess.exithandler (child_process.js:217:12)
    at emitTwo (events.js:106:13)
    at ChildProcess.emit (events.js:194:7)
    at maybeClose (internal/child_process.js:899:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:226:5)
```

**Comment by jakubzitny** [comment]
Hi @j-sharrock, thanks for reporting this error, sorry to see it. We've had a similar problem in #1742. 

This is a bug in our detection of Sketch.app. Please move Sketch to `/Applications` and rename it to `Sketch.app` and it should fix itself. In the meantime we will fix the detection on our side.

Please, let us know if it helped. If not, you can download the plugin manually [here](https://avocode-media.s3.amazonaws.com/download/sketch-plugin-new/47.1.3/sketch-plugin-new-mac-47.1.3.sketchplugin.zip).

**Comment by jakubzitny** [comment]
Hi @j-sharrock, we've released 2.17.1 few days back. This should be fixed there.
